src/mobile/src/ui/components/TopBar.js

```diff
@@ -233,6 +232,22 @@ class TopBar extends Component {
         }
     }
 
+    /**
+     * Returns padding styling dependent on device type
+     *
+     * @method getTopbarPadding
+     * @returns {object}
```

Comment:
The return type should be of type `any` since there's also a possibility that this function could return an `undefined` value (In case of iPhoneX).